src/workloads/query/CollScanPredicateSelectivityLarge.yml

```diff
@@ -0,0 +1,26 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests the performance of conjunctive collection scan queries where the order of
+  predicates matters due to selectivity of the predicates.
+
+LoadConfig:
+  Path: ../../phases/query/CollScanPredicateSelectivity.yml
+  Parameters:
+    Database: CollScanPredicateSelectivityLarge
+    DocumentCount: 1e6
+    Repeat: 10
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - standalone
+      - standalone-all-feature-flags
+      - standalone-classic-query-engine
+      - standalone-heuristic-bonsai
+      - standalone-sampling-bonsai
+      - standalone-sbe
+    branch_name:
+      $gte:
+        v7.3
```

Comment:
Does this workload need to be backported to v7.3 branch? That being said I don't think we run any of the variants listed above on our release branches.

Reply:
No need to backport this. I'll adjust the predicate to make it more clear that this should only run on master.